pages/index.tsx

```diff
@@ -1,13 +1,20 @@
+/**
+ *  index.tsx
+ *
+ *  @copyright 2024 Digital Aid Seattle
+ *
+ */
+import React, { useState } from 'react';
+import { authService } from '../src/services/authService';
```

Comment:
This and the useRouter are not being used. Are you including them because they will be used in the future?